### PR TITLE
Add acceptance test for data_rescue_image

### PR DESCRIPTION
**Pull request by meyskens**
Signed-off-by: Maartje Eyskens <maartje@eyskens.me>

### CODE REVIEW

File: provider/data_rescue_image_test.go [hunk 1]
```diff
@@ -52,3 +52,38 @@ func TestDataRescueImage(t *testing.T) {
 		},
 	})
 }
+
+func TestDataRescueImageAcceptance(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers:  testAccProviders,
+		IsUnitTest: false,
```
Comment (rporres) [inline provider/data_rescue_image_test.go]: nit: Is this needed?

**Comment by rporres** [comment]
Looks good, just a little of nit-picking :)

File: provider/data_rescue_image_test.go (hunk 1 above)
Comment (meyskens) [reply provider/data_rescue_image_test.go]: I was doubting the same but kept it as a verbose indication that this is definitely not a unit test like the one above

Comment (rporres) [reply provider/data_rescue_image_test.go]: Fair enough

**Pull request merged**